src/ui/components/ProtocolViewer.tsx

```diff
@@ -74,7 +74,8 @@ const chunkedRequests = (requests: RequestSummary[]): RequestSummaryChunk[] => {
         startedAt: 0,
       },
     }
-  ).chunks;
+  );
+  return [...chunks.chunks, chunks.current];
```

Comment:
So this is handling the fact that we don't initialize the `chunks` array with our first `current` and so each time we make a new `current` we push the _previous_ one to the array?

I wonder if we could write this a little more clearly by restructuring the code a bit?
```js
const mergedRequests = (requests: RequestSummary[]): RequestSummaryChunk[] => {
  let current = {
    count: 0,
    ids: [],
    method: "",
    pending: false,
    errored: false,
    startedAt: 0,
  };

  const merged = [current];

  requests.forEach((request: RequestSummary) => {
    if (
      current.method === fullMethod(request) &&
      current.pending === request.pending &&
      current.errored === request.errored
    ) {
      current.count++;
      current.ids.push(request.id);
    } else {
      current = {
        count: 1,
        ids: [request.id],
        errored: request.errored,
        method: fullMethod(request),
        pending: request.pending,
        startedAt: request.recordedAt,
      };

      merged.push(current);
    }
  });

  return merged;
};
```

This is a nit so feel free to ignore.